src/NuGet.Services.AzureSearch/AuxiliaryFiles/AuxiliaryFileClient.cs

```diff
@@ -41,6 +41,27 @@ public class AuxiliaryFileClient : IAuxiliaryFileClient
 
         private ICloudBlobContainer Container => _lazyContainer.Value;
 
+        public async Task<DownloadData> LoadDownloadDataAsync()
+        {
+            var result = await LoadAuxiliaryFileAsync(
+                _options.Value.AuxiliaryDataStorageDownloadsPath,
+                etag: null,
+                loadData: loader =>
+                {
+                    var downloadData = new DownloadData();
+
+                    Downloads.Load(
+                        name: null,
+                        loader: loader,
+                        addCount: (id, version, count) => downloadData.SetDownloadCount(id, version, count));
```

Comment:
Can this be simplified?

```suggestion
                        addCount: downloadData.SetDownloadCount;
```

Reply:
No, the `Download` type operates with `int` and `DownloadData` uses `long`. But I will change this method to use `long` so this can work.